code/web/src/modules/survey/Survey.js

```diff
@@ -70,6 +72,9 @@ class Survey extends PureComponent {
 
         toggleSelection = (id) => {
           this.props.toggleSelection(id, this.props.survey.products)
+          this.setState(prevState=> ({
+            selectCount: prevState.selectCount + 1
+          }))
```

Comment:
I could be missing this or looking in the wrong commit but does this count get reset when the survey is cleared?

Reply:
Because selectCount is in local state, when a new Survey is instantiated, it is made with a selectCount of 0.

Reply:
Perfect! Makes sense!